maintainers/github-teams.json

```diff
@@ -912,17 +911,6 @@
     },
     "name": "SDL"
   },
-  "security": {
```

Comment:
This is interesting. It looks like the token in CI doesn't see this team as having access to Nixpkgs. It's indeed not a collaborator, but has organisation-based access: 

<img width="911" height="283" alt="Image" src="https://github.com/user-attachments/assets/8258a242-d30b-4cea-a8ba-637fa025d282" />

I think we can just ignore this for now, as the team doesn't have a corresponding `lib.teams` anyways.